From: Ole Streicher <olebole@debian.org>
Date: Sat, 24 Dec 2016 13:47:37 +0100
Subject: Avoid privacy breach

use standard fonts instead of loading something from google
use local jquery from sphinx
---
 doc/source/themes/scikit-image/layout.html | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/doc/source/themes/scikit-image/layout.html b/doc/source/themes/scikit-image/layout.html
index 945dc01..79edabe 100644
--- a/doc/source/themes/scikit-image/layout.html
+++ b/doc/source/themes/scikit-image/layout.html
@@ -15,7 +15,7 @@
   {%- set titlesuffix = "" %}
 {%- endif %}
 {%- macro script() %}
-    <script src="https://code.jquery.com/jquery-latest.js"></script>
+    <script src="file:///usr/share/javascript/jquery/jquery.js"></script>
     <script src="{{ pathto('_static/', 1) }}js/bootstrap.min.js"></script>
     <script src="{{ pathto('_static/', 1) }}js/togglebutton.js"></script>
     <script type="text/javascript">
@@ -35,7 +35,6 @@
     <link rel="stylesheet" href="{{ pathto('_static/pygments.css', 1) }}" type="text/css" />
     <link href="{{ pathto('_static/', 1) }}css/bootstrap.min.css" rel="stylesheet" type="text/css">
     <link href="{{ pathto('_static/', 1) }}css/custom.css" rel="stylesheet" type="text/css">
-    <link href="https://fonts.googleapis.com/css?family=Raleway" rel="stylesheet" type="text/css">
     {%- for cssfile in css_files %}
         <link rel="stylesheet" href="{{ pathto(cssfile, 1) }}" type="text/css" />
     {%- endfor %}
